src/averageAges.js

```diff
@@ -14,14 +14,18 @@
  *
  * @return {number}
  */
+const calculateAverageAge = (people) =>
+  people.reduce((acc, item) => acc + item.died - item.born, 0) / people.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const arrayOfMen = people.filter((person) => {
```

Comment:
It is better to not specify the type of variable in it's name, so it can be just `men`